### PR TITLE
update changelog for 7.10.0 release

### PR DESCRIPTION
**Pull request by chliangGoogle**
Also notice the podspec wasn't updated from last release. Is this ideal?

### CODE REVIEW

**Comment by ncooke3** [comment]
> Also notice the podspec wasn't updated from last release. Is this ideal?

7.9.0 was SPM only so this is expected. See https://github.com/google/GoogleUtilities/blob/main/CHANGELOG.md#790-swift-pm

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 7.10.0 (Swift PM)
```
Comment (ncooke3) [inline CHANGELOG.md:1]:
```suggestion
# 7.10.0
```

**Review by ncooke3 — APPROVED** [review]
One changelog nit since I believe this release is not package-manager specific. Otherwise, lgtm 👍

File: CHANGELOG.md (hunk 1 above)
Comment (chliangGoogle) [reply CHANGELOG.md:1]: Thanks for confirming. I was wondering that.

Comment (ncooke3) [reply CHANGELOG.md]: yep, for sure!

**Pull request merged**